Gems/EditorPythonBindings/Code/Source/PythonProxyObject.cpp

```diff
@@ -846,7 +846,7 @@ namespace EditorPythonBindings
                         {
                             return ConstructPythonProxyObjectByTypename(behaviorClassName, pythonArgs);
                         });
-                        PythonSymbolEventBus::QueueBroadcast(&PythonSymbolEventBus::Events::LogClassWithName, subModuleName, behaviorClass, properSyntax);
+                        PythonSymbolEventBus::QueueBroadcast(&PythonSymbolEventBus::Events::LogClassWithName, subModuleName, behaviorClass, syntaxName.value());
```

Comment:
question: Would be great if either @AMZN-AlexOteiza or @jackalbe could take a look at this to verify

Reply:
I just do not see the value of the change, unless I am missing something